Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
If it's literally never going to be fixed upstream I'm a bit apprehensive about carrying it. On the flip side if we're never gonna see any new versions then: meh.